rasa/utils/tensorflow/constants.py

```diff
@@ -40,7 +40,7 @@
 DROP_RATE_LABEL = "drop_rate_label"
 CONSTRAIN_SIMILARITIES = "constrain_similarities"
 
-WEIGHT_SPARSITY = "weight_sparsity"
```

Comment:
you cannot simply remove it, you need to deprecate it where you recalculate CONNECTION_DENSITY

Reply:
Ahhh... right. Will do that.